tests/expectations/metrics/test_map_metric.py

```diff
@@ -349,6 +332,44 @@ def test_get_aggregate_count_aware_metric_dependencies(basic_spark_df_execution_
     )
 
 
+def test_get_aggregate_count_aware_metric_dependencies(basic_spark_df_execution_engine):
```

Comment:
Is this a duplicate?

Reply:
Shoot you're right. Thanks for catching that @anthonyburdi